### PR TITLE
1.9-doc-tweak

### PR DESCRIPTION
**Pull request by NigelBreslaw**
Fix the element headings to match live-preview.
Simplify VS Code setup

### CODE REVIEW

File: docs/astro/src/content/docs/guide/tooling/vscode.mdx [hunk 1]
```diff
@@ -22,32 +22,17 @@ We support many other tools and editors, see [here](https://github.com/slint-ui/
...
 1. **Install VS Code.**   
 Download it [here](https://code.visualstudio.com).
 
 2. **Install the Slint extension.**  
 Find it [here](https://marketplace.visualstudio.com/items?itemName=Slint.slint).
 
-3. **Create a new project based on a Slint template.**  
-This is done via the command palette (CMD+Shift+P).
+3. **Create a new project based on a Slint template.** 
+This is done via the command palette (CTRL+Shift+P) or on MacOS (CMD+Shift+P).
```
Comment (ogoffart) [inline docs/astro/src/content/docs/guide/tooling/vscode.mdx:35]: Ideally it should also state what to type in the command palette, in text.
(Screenshot alone is not accessible and cannot be copy-pasted)

**Comment by ogoffart** [comment]
The commit seems to remove the screenshot images while they are still in the mdx file. Am I missing something?

**Comment by NigelBreslaw** [comment]
They are not removed. I updated them because one was in dark mode, while the others were light and the other was showing a ton of plugins I have. So I turned them off to clear up the image.

**Comment by ogoffart** [comment]
Ah Ok, the Github UI is just confusing.

**Review by ogoffart — APPROVED** [review]

**Pull request merged**